reo/tests/test_case.py

```diff
@@ -0,0 +1,247 @@
+import json
+from django.test import TestCase
+from tastypie.test import ResourceTestCaseMixin
+from unittest import skip
+
+
+class TestCase(ResourceTestCaseMixin, TestCase):
```

Comment:
please rename and clean up this test (or delete it?) it is not clear what it is testing

Reply:
This was copied from the private branch, and it's not clear that we need it here.  I've deleted it via commit a3fbd36015eeb254ffe0ddcb780af6a78d5fba5c.